From 46684d386c7b99056b2cbda6bab1c489b21dfdb9 Mon Sep 17 00:00:00 2001
From: Varun Vasudev <vvasudev@apache.org>
Date: Tue, 27 Oct 2015 20:56:00 +0530
Subject: [PATCH 0981/2748] YARN-4302. SLS not able start due to NPE in
 SchedulerApplicationAttempt. Contributed by Bibin
 A Chundatt.

(cherry picked from commit c28e16b40caf1e22f72cf2214ebc2fe2eaca4d03)
(cherry picked from commit c5bf1cb7af33d93cff783b00d945eb3be66c3c5e)

Change-Id: Ie957aadd0cfdfc7d9d9769a17bae42db320a3ee5
---
 .../sls/scheduler/ResourceSchedulerWrapper.java    |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/hadoop-tools/hadoop-sls/src/main/java/org/apache/hadoop/yarn/sls/scheduler/ResourceSchedulerWrapper.java b/hadoop-tools/hadoop-sls/src/main/java/org/apache/hadoop/yarn/sls/scheduler/ResourceSchedulerWrapper.java
index 3ac3382..0020933 100644
--- a/hadoop-tools/hadoop-sls/src/main/java/org/apache/hadoop/yarn/sls/scheduler/ResourceSchedulerWrapper.java
+++ b/hadoop-tools/hadoop-sls/src/main/java/org/apache/hadoop/yarn/sls/scheduler/ResourceSchedulerWrapper.java
@@ -920,7 +920,7 @@ public String moveApplication(ApplicationId appId, String newQueue)
   @LimitedPrivate("yarn")
   @Unstable
   public Resource getClusterResource() {
-    return null;
+    return super.getClusterResource();
   }
 
   @Override
-- 
1.7.9.5

